spec/migrations_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'database_helper'
+require 'models'
```

Comment:
Didn't see this line require 'models'` in upstream, why do we need it here?
https://github.com/chaadow/active_record-acts_as/blob/b10bc264e73f679906de376e350236ccf768cee2/spec/migrations_spec.rb#L2

Reply:
the running of this test, should we not include this line, really depends on whether during initialization (before the test is being run), the file `models.rb` inside `spec` module has been executed and the definition of `initialize_schema` inside that file is defined already

I tested in my local, this spec file failed most of the time if this line is not included, and hence I need to add this line to ensure that everything inside `spec/models.rb` is fetched and analysed first before one of its functions inside is being used inside this test file here

Reply:
Sounds flaky, will this solve our own flaky issues on main CM repo?